test-suite/js/src/test/scala/org/scalajs/testsuite/library/LinkTimePropertyTest.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import scala.scalajs.js
+import scala.scalajs.LinkingInfo
+import scala.scalajs.LinkingInfo._
+
+import org.junit.Test
+import org.junit.Assert._
+
+import org.scalajs.testsuite.utils.Platform
+
+class LinkTimePropertyTest {
+  @Test def linkTimeProperty(): Unit = {
```

Comment:
I believe this is already covered by `LinkingInfoTest`, isn't it?

Reply:
Right... I somehow duplicated the same test even though the file is next to each other 🤔 removed